COLLABORATOR_GUIDE.md

```diff
@@ -411,6 +412,22 @@ For new modules that involve significant effort, non-trivial additions to
 Node.js or significant new capabilities, an [Enhancement Proposal][] is
 recommended but not required.
 
+### Additions to N-API
+
+N-API provides an ABI stable API that we will have to support in future
+versions without the usual option to modify or remove existing APIs on
+SemVer boundaries. Therefore, additions need to be managed carefully.
+
+This
+[guide](https://github.com/nodejs/node/blob/master/doc/guides/adding-new-napi-api.md)
+outlines the requirements and principals that we should follow when
+approving and landing new N-API APIs (any additions to node_api.h and
+node_api_types.h).
+
+Two of the key requirements are that additions be landed as
+experimental and then only promoted out of experimental with agreement
+from the [n-api team](https://github.com/orgs/nodejs/teams/n-api). 
```

Comment:
I'd prefer this paragraph be removed. Leaving it in means we have to update things in two places if anything changes. More importantly, if this paragraph has a subtle-but-meaningful difference from what's in the linked doc, it may be unclear as to which document carries more authority.

I understand the motivation to summarize, but if someone is adding an N-API  addition, they should (and no doubt will, if they see this section) read the linked doc. If they aren't adding an N-API addition, then this paragraph is not useful to them.

Reply:
Ok, will remove